fischkapp/src/components/Header.tsx

```diff
@@ -0,0 +1,14 @@
+import { StyledHeader } from "./styles/Header.styled";
+export const Header = () => {
+  return (
+    <StyledHeader>
+      <div className="header__logo">
+        <h1 className="header__logoName">FLASHCARDS</h1>
+        <p className="header__cardsNumber">0</p>
+      </div>
+      <button>
+        <p></p>
```

Comment:
**nitpick:** We should use SVG icon here. You should be able to change your icon in button when needed. Now, I have no idea what icon is used in this button